babel.config.js

```diff
@@ -38,6 +38,7 @@ const config = {
             },
         ],
     ],
+    sourceType: 'unambiguous',
```

Comment:
I don't entirely understand why this was needed, but without it, the web app would fail to run with an error about `exports` being undefined

Reply:
It has something to do with require, module.exports usage with webpack